spec/io/sql_data_source_spec.rb

```diff
@@ -62,5 +62,24 @@
         }.to raise_error(ArgumentError)
       end
     end
+
+    context 'with path to sqlite3 file' do
+      it 'returns a dataframe' do
+        result = Daru::IO::SqlDataSource.make_dataframe(db_name, query)
```

Comment:
Updated the docs, but I was unsure if this was the correct way to do so:
https://monosnap.com/file/fgILJV5wPNLvpbeXEsIs7f0GRdLYxq